From 2dfdac5288cb58bfe2891195e9c90249c81844a7 Mon Sep 17 00:00:00 2001
From: Levi Yun <ppbuk5246@gmail.com>
Date: Thu, 9 Nov 2023 16:08:22 +0000
Subject: [PATCH 33/40] maple_tree: change return type of mas_split_final_node
 as void.

mas_split_final_node always returns true and there is no usage
for checking return value.

change return type as void.

Signed-off-by: Levi Yun <ppbuk5246@gmail.com>
---
 lib/maple_tree.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index 1e9add506..374707b51 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -3146,7 +3146,7 @@ static inline void mas_destroy_rebalance(struct ma_state *mas, unsigned char end
  * @mas: The maple state
  * @height: The height of the tree in case it's a new root.
  */
-static inline bool mas_split_final_node(struct maple_subtree_state *mast,
+static inline void mas_split_final_node(struct maple_subtree_state *mast,
 					struct ma_state *mas, int height)
 {
 	struct maple_enode *ancestor;
@@ -3170,7 +3170,6 @@ static inline bool mas_split_final_node(struct maple_subtree_state *mast,
 	mast->l->node = ancestor;
 	mab_mas_cp(mast->bn, 0, mt_slots[mast->bn->type] - 1, mast->l, true);
 	mas->offset = mast->bn->b_end - 1;
-	return true;
 }
 
 /*
-- 
2.43.0.232.ge79552d197

